packages/integration-sdk-cli/src/generator/template/test/README.md.hbs

```diff
@@ -0,0 +1,99 @@
+## Notes on `test/config.ts`
```

Comment:
We should just link out to docs and get rid of this. We'd never update this in every repo 😬